### PR TITLE
refactor: update search ts template

### PR DESCRIPTION
**Pull request by yukun-dong**

### CODE REVIEW

File: templates/ts/m365-message-extension/teamsapp.yml.tpl [hunk 1]
```diff
@@ -48,7 +48,7 @@ provision:
           # variable before ARM deployment.
           parameters: ./infra/azure.parameters.json
           # Required when deploying ARM template
-          deploymentName: Create-resources-for-tab
+          deploymentName: Create-resources-for-mxt
```
Comment (kuojianlu) [inline templates/ts/m365-message-extension/teamsapp.yml.tpl:51]:
```suggestion
          deploymentName: Create-resources-for-me
```

**Comment by kuojianlu** [comment]
- JS template will be in a separate PR?
- Since codes are moved to `src`, I suppose there's a need to update the `dev` script in `package.json`.

**Comment by yukun-dong** [comment]
> * JS template will be in a separate PR?

Yes, I will create another PR to update js template.

File: templates/ts/m365-message-extension/teamsapp.yml.tpl (hunk 1 above)
Comment (kuojianlu) [reply templates/ts/m365-message-extension/teamsapp.yml.tpl:51]: Or `-for-me-search` to keep consistent with other me samples.

**Review by kuojianlu — APPROVED** [review]

File: templates/ts/m365-message-extension/teamsapp.yml.tpl (hunk 1 above)
Comment (yukun-dong) [reply templates/ts/m365-message-extension/teamsapp.yml.tpl:51]: I think it's a bit confusing to abbreviate "message extension" to "me". Others may not know what "me" stand for.
@MuyangAmigo @summzhan What are your opinions on this?

Comment (kuojianlu) [reply templates/ts/m365-message-extension/teamsapp.yml.tpl:51]: I see message-extension uses `-mxt` and message-extension-action uses `-me-action`...

**Review by summzhan — APPROVED** [review]

**Review by kuojianlu — DISMISSED** [review]

File: templates/ts/m365-message-extension/teamsapp.yml.tpl (hunk 1 above)
Comment (yukun-dong) [reply templates/ts/m365-message-extension/teamsapp.yml.tpl]: @hund030 will use template maintenance tools to unify these later.

**Review by kuojianlu — APPROVED** [review]

**Pull request merged**